File: harvwstbackend/harvwstbackend/Modles/HarvestEntity.cs
namespace harvestbackend.Modles
{
    public class HarvestEntity
    {
        public int Id { get; set; }
        public DateTime HarvestDate { get; set; }
        public string SuperLeafWeight { get; set; }
        public string NormalLeafWeight { get; set; }
        public string TransportMethod { get; set; }
        public string PaymentMethod { get; set; }
    }

}

File: tfsms_app_frontend/lib/models/harvest_entity.dart
class HarvestEntity {
  final DateTime date;
  final double superLeafWeight;
  final double normalLeafWeight;
  final String transportMethod;
  final String paymentMethod;

  HarvestEntity({
    required this.date,
    required this.superLeafWeight,
    required this.normalLeafWeight,
    required this.transportMethod,
    required this.paymentMethod,
  });

  Map<String, dynamic> toJson() {
    return {
      'date': date.toIso8601String(),
      'superLeafWeight': superLeafWeight,
      'normalLeafWeight': normalLeafWeight,
      'transportMethod': transportMethod,
      'paymentMethod': paymentMethod,
    };
  }
}
